packages/mobile/ios/Podfile

```diff
@@ -0,0 +1,80 @@
+  pod 'DoubleConversion', :podspec => '../node_modules/react-native/third-party-podspecs/DoubleConversion.podspec'
+  pod 'Folly', :podspec => '../node_modules/react-native/third-party-podspecs/Folly.podspec'
+  pod 'glog', :podspec => '../node_modules/react-native/third-party-podspecs/GLog.podspec'
+
+  # React Native Libraries
+  # pod 'RNFBApp', :path => '../node_modules/@react-native-firebase/app'
+  # pod 'RNFBAuth', :path => '../node_modules/@react-native-firebase/auth'
+  # pod 'RNFBDatabase', :path => '../node_modules/@react-native-firebase/database'
+  pod 'RNFirebase', :path => '../../../node_modules/react-native-firebase/ios'
+  pod 'RNFS', :path => '../../../node_modules/react-native-fs'
+  pod 'react-native-geth', :path => '../../../node_modules/react-native-geth'
+  pod 'RNSVG', :path => '../../../node_modules/react-native-svg'
+  pod 'RNAnalytics', :path => '../../../node_modules/@segment/analytics-react-native'
+  pod 'RNAnalyticsIntegration-Firebase', :path => '../../../node_modules/@segment/analytics-react-native-firebase'
+  pod 'react-native-config', :path => '../../../node_modules/react-native-config'
+  pod 'react-native-contacts', :path => '../../../node_modules/react-native-contacts'
+  pod 'RNDeviceInfo', :path => '../../../node_modules/react-native-device-info'
+  pod 'react-native-keep-awake', :path => '../../../node_modules/react-native-keep-awake'
+  pod 'RNLanguages', :path => '../../../node_modules/react-native-languages'
+  pod 'react-native-mail', :path => '../../../node_modules/react-native-mail'
+  pod 'RNScreens', :path => '../../../node_modules/react-native-screens'
+  pod 'SentryReactNative', :path => '../../../node_modules/react-native-sentry'
+  pod 'react-native-splash-screen', :path => '../../../node_modules/react-native-splash-screen'
+  pod 'react-native-version-check', :path => '../../../node_modules/react-native-version-check'
+  pod 'RNRandomBytes', :path => '../../../node_modules/react-native-secure-randombytes'
+  pod 'react-native-udp', :path => '../../../node_modules/react-native-udp'
+  pod 'react-native-netinfo', :path => '../../../node_modules/@react-native-community/netinfo'
+  pod 'RNShare', :path => '../../../node_modules/react-native-share'
+  pod 'react-native-camera', :path => '../../../node_modules/react-native-camera'
+  pod 'RNGestureHandler', :path => '../node_modules/react-native-gesture-handler'
+  pod 'CeloBlockchain', :path => '../../../node_modules/@celo/client/CeloBlockchain.podspec'
+  pod 'RNSecureKeyStore', :path => '../../../node_modules/react-native-secure-key-store/ios'
+
+  pod 'Firebase/Core', '~> 5.20.2'
+  pod 'GoogleUtilities', '~> 5.3.7'
+  # pod 'Segment-Firebase', '~> 2.4.0'
+
+  target 'celoTests' do
+    inherit! :search_paths
+  end
+end
+
+post_install do |installer|
+  installer.pods_project.targets.each do |target|
+    target.build_configurations.each do |config|
+      config.build_settings['ENABLE_BITCODE'] = 'NO'
```

Comment:
Geth framework doesn't support bitcode. Not really a deal breaker, I tend to disable bitcode usually as it complicates crash reporting a little bit (you have to download the symbols after uploading the app to App Store Connect to be able to symbolicate stacktraces, It can (should) be automated anyway but takes longer in the CI pipeline).